### PR TITLE
feat: Steal action

### PR DESCRIPTION
**Pull request by Panzzzerfaust**

### CODE REVIEW

File: KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs [hunk 1]
```diff
@@ -251,6 +253,38 @@ private async Task HandleKillCharacter(long chatId, string[] data)
...
+            player.TelegramMetadata.CardMessageIds.AsParallel().WithDegreeOfParallelism(5).ForAll(async id => await _botClient.TryDeleteMessage(chatId, id));
+            player.TelegramMetadata.CardMessageIds.Clear();
+
+            var characterName = data[2];
+            var targetName = data[3];
+
+            lobby.CharacterDeck.Find(x => x.Name == targetName).Effect = CharacterEffect.Robbed;
+            player.GameActions.Remove(GameAction.Steal);
+
+            await _playerProvider.SavePlayer(player);
```
Comment (AntonArtomovNM) [inline KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs:283]: Should we use .UpdatePlayer() instead?

Comment (Panzzzerfaust) [reply KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs:283]: ok)

File: KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs [hunk 2]
```diff
@@ -359,6 +393,30 @@ private async Task SendKillCharacterKeyboard(long chatId, Lobby lobby, Player pl
             await _playerProvider.UpdatePlayer(player.Id, p => p.TelegramMetadata, player.TelegramMetadata);
         }
 
+        private async Task SendStealCharacterKeyboard(long chatId, Lobby lobby, Player player, string characterName, string gameAction)
```
Comment (AntonArtomovNM) [inline KCAA/Services/TelegramApi/TelegramUpdateHandlers/TelegramCallbackQueryHandler.cs:396]: Method looks almost the same as SendKillCharacterKeyboard, maybe we should combine them?

**Review by AntonArtomovNM — APPROVED** [review]

**Comment by AntonArtomovNM** [comment]
["Steal" action](https://trello.com/c/aWvxKKUe/43-steal-action)

**Pull request merged**